core/src/components/button/test/spinner/button.e2e.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ionic-md', 'ionic-ios'] }).forEach(({ config, screenshot, title }) => {
```

Comment:
Since the functionality doesn't change, we don't need to test for all the modes. I also realize that we don't need to test RTL since functionality doesn't change.

I suggest only testing `ionic-md`. Make sure to remove the snapshots for `ionic-ios` and RTL.

```suggestion
configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ config, screenshot, title }) => {
```